### PR TITLE
feat(galaxy): migrate galaxy to monorepo

### PR DESCRIPTION
**Pull request by tmastrom**
This PR 
* updates the `@scalar/galaxy` to build with our rollup tooling. 
* adds a playground example in `@scalar/galaxy` to serve the galaxy spec with the `@scalar/mock-server`
*  creates a Dockerfile and cloudbuild script to deploy the the `@scalar/mock-server` example (the mock-server and galaxy playground examples are the same so we can deploy either one)

### CODE REVIEW

File: packages/galaxy/rollup.config.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      dest: 'dist',
+      rename: 'latest.yaml',
+    },
+  ],
+  options: {
+    input: entries,
+  },
+})
+
+await addPackageFileExports({ entries })
```
Comment (tmastrom) [inline packages/galaxy/rollup.config.ts:23]: I am having issues with this overwriting the the .json and .yaml file exports in the package.json

File: packages/mock-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+COPY turbo.json .
+COPY packages ./packages
+RUN pnpm install
+RUN pnpm turbo \
+    run build \
+    --filter='./packages/*' \
+    # … don’t build too many in parallel, we don’t want to hit a memory limit.
+    --concurrency=2
+
+FROM base AS builder
```
Comment (geoffgscott) [inline packages/mock-server/Dockerfile:27]: We can just use base builder here.

File: packages/galaxy/rollup.config.ts (hunk 1 above)
Comment (geoffgscott) [reply packages/galaxy/rollup.config.ts:23]: That is precisely what it does. In your case I would suggest just manually provision the exports field.

**Review by geoffgscott — APPROVED** [review]
Looks good.

**Review by marclave — CHANGES_REQUESTED** [review]
we should add a changeset to this so we know the deploys to npm are working

https://www.npmjs.com/package/@scalar/galaxy

**Pull request merged**